app/src/main/java/com/yeonae/chamelezone/view/login/JoinFragment.kt

```diff
@@ -28,6 +23,7 @@ class JoinFragment : Fragment(), JoinContract.View {
     override fun showMessage(message: String) {
         Toast.makeText(context, message, Toast.LENGTH_LONG)
             .show()
+        (activity as LoginActivity).back(JoinFragment())
```

Comment:
```suggestion
        (activity as LoginActivity).back(this)
```
해야한다면 이게 맞고, 애초에 개선이 필요해보이긴 합니다.
`JoinFragment()` 이렇게하면 새로만든걸 넣어주는거에요 new해서요

Reply:
수정했습니당